Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionURLSchemeHandlerCocoa.mm

```diff
@@ -66,11 +66,12 @@
         URL frameDocumentURL = task->frameInfo().request().url();
         URL requestURL = task->request().url();
 
-        if (task->frameInfo().request().url().isEmpty() || task->frameInfo().request().url().isAboutBlank()) {
+        Ref frameInfo = task->frameInfo();
```

Comment:
We could also move `Ref frameInfo = task->frameInfo();` above `URL frameDocumentURL = task->frameInfo().request().url();` so it can be used there too. But it's not a big deal. Up to you.

Reply:
Oh good point.